app/views/content/funding-and-support/if-youre-a-parent-or-carer.md

```diff
@@ -71,15 +71,15 @@ To be eligible, you need to:
 
 The Parents’ Learning Allowance could help you with your learning costs if you’re a full-time student with children.
 
-You could be eligible for between £50 and £1,963 a year, depending on your household income.
+You could be eligible for between $finance_parentslearningallowance_lowpayment$ and $finance_parentslearningallowance_maxpayment$ a year, depending on your household income.
```

Comment:
finance_parentslearningallowance_low (although does it make more sense that this ends in min if the other one is max?)

finance_parentslearningallowance_max